Configuration/AlCa/python/autoCondModifiers.py

```diff
@@ -96,3 +96,32 @@ def autoCond2017ppRef5TeV(autoCond):
                                          L1Menu_2017ppRef5TeV)
     autoCond.update(GlobalTag2017ppRef5TeV)
     return autoCond
+
+def autoCondRelValForRun2(autoCond):
+
+    GlobalTagRelValForRun2 = {}
+    L1GtTriggerMenuForRelValForRun2 =    ','.join( ['L1Menu_Collisions2015_25nsStage1_v5' , "L1GtTriggerMenuRcd",             connectionString, "", "2023-01-28 12:00:00.000"] )
+    L1TUtmTriggerMenuForRelValForRun2 =  ','.join( ['L1Menu_Collisions2018_v2_1_0-d1_xml' , "L1TUtmTriggerMenuRcd",           connectionString, "", "2023-01-28 12:00:00.000"] )
+
+    for key,val in autoCond.items():
+        if 'run2_data' in key or 'run2_hlt' in key:
```

Comment:
I think `if 'run2_data' in key` will generate unwanted symbolic GTs since in autoCond we also have:
 - run2_data_HEfail --> will generate `run2_data_HEfail_relval` 
 - run2_data_promptlike_hi --> will generate `run2_data_promptlike_hi_relval`